Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.11-alpine
+
+RUN pip install requests
+
+ADD . /src
+
+# UDP
+EXPOSE 1812
```

Comment:
That would certianly be more correct tho I thought I tested with udp port exporting but maybe I missed something.

It is possible it also had to do with how the proxy handles source of the udp packets.